operator/duties/attester.go

```diff
@@ -81,11 +81,9 @@ func (h *AttesterHandler) HandleDuties(ctx context.Context) {
 				h.fetchFirst = false
 				h.indicesChanged = false
 				h.processFetching(ctx, currentEpoch, slot)
-				// TODO: (Alan) genesis support
-				//h.processExecution(currentEpoch, slot)
+				h.processExecution(currentEpoch, slot)
 			} else {
-				// TODO: (Alan) genesis support
-				//h.processExecution(currentEpoch, slot)
+				h.processExecution(currentEpoch, slot)
```

Comment:
I thought we are not executing this.. only in committee.go , we can leave the 
// TODO: (Alan) genesis support
so we remember a fork might be needed here.

Reply:
oh I see its only aggregator now.